vars/execSmokeTest.groovy

```diff
@@ -105,12 +105,16 @@ private def deployEnvironment(
     }
 
     // Deploy the other service sets
-    for (serviceSet in ocDeployerServiceSets.split(',')) {
+    ocDeployerServiceSets.split(',').eachWithIndex { serviceSet, i ->
         def set = serviceSet // https://jenkins.io/doc/pipeline/examples/#parallel-multiple-nodes
+        // Deploy the tested service with scaled resources and other services with default resources
```

Comment:
I'd prefer if we added a boolean option like `scaleFirstSetOnly` (false by default) to enable this behavior because otherwise we're changing how this has worked in the past and I'm not sure if it is going to impact anyone

Reply:
added `scaleFirstSetOnly`